apps/gitness/src/pages/repo/repo-branch-list.tsx

```diff
@@ -30,19 +36,49 @@ export function ReposBranchesListPage() {
   const repoRef = useGetRepoRef()
 
   const { currentPage, previousPage, nextPage, handleClick } = usePagination(1, totalPages)
+  const { data: repoMetadata } = useFindRepositoryQuery({ repo_ref: repoRef })
 
-  const { isLoading, data: brancheslistData } = useListBranchesQuery({
-    queryParams: { page: currentPage, limit: 20, sort: 'date', order: 'desc', include_commit: true },
+  const {
+    isLoading,
+    data: brancheslistData,
+    isError
+  } = useListBranchesQuery(
+    {
+      queryParams: { page: currentPage, limit: 20, sort: 'date', order: 'desc', include_commit: true },
+      repo_ref: repoRef
+    },
+    {
+      onError: (error: ListBranchesErrorResponse) => {
+        console.error('Error BranchList', error)
+      }
+    }
+  )
+
+  const { data: getBranchDivergence, mutate } = useCalculateCommitDivergenceMutation({
```

Comment:
`getBranchDivergence` sounds like a method name, that returns `branchDivergence`. Instead rename `getBranchDivergence` to just `branchDivergence`.